site/docs/components/progress-bar.mdx

```diff
@@ -38,11 +39,8 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 - Static process or loading progress:
     - **Static**: a static progress bar shows current progress through a process that will not progress without user interaction.
     - **Loading**: a loading progress bar shows current progress through a loading process that the system will progress by itself without user interaction. We use animation to show that the system is operating.
-- Props:
-    - Min
-    - Max
-    - Initial value
-    - Infinite/indeterminate
+- Percentage (33%) or fraction (1/10)
```

Comment:
Do we actually want to support fractions as labels, or was this from when the value prop could accept a fraction?

Reply:
It's from the original docs but the original docs were more specific in the implementation. I see the docs as design guidelines (as in "it is common or acceptable to use a fraction as a label"). So where are implementation is extremely flexible (people can put whatever they want in the label), the docs are there to provide guidance for common uses.